CHANGELOG.md

```diff
@@ -0,0 +1,60 @@
+## Widget (2024-05-08)
```

Comment:
As we have talked, it would be great to specify version for the release.

Reply:
We can't. We didn't increase client's version before. For upcoming releases, it will have version as well as you can see in the template.

Reply:
I think client versions could not help us here. We need to have embedded version here.